epoxy-paging3/src/main/java/com/airbnb/epoxy/paging3/PagingEpoxyController.java

```diff
@@ -0,0 +1,279 @@
+package com.airbnb.epoxy.paging3;
+
+import com.airbnb.epoxy.EpoxyController;
+import com.airbnb.epoxy.EpoxyModel;
+import com.airbnb.epoxy.EpoxyViewHolder;
+import com.airbnb.epoxy.paging3.PagedListEpoxyController;
+
+import java.util.Collections;
+import java.util.List;
+
+import androidx.annotation.CallSuper;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.paging.PagedList;
+import androidx.paging.PagedList.Callback;
+import androidx.paging.PagedList.Config;
+
+/**
+ * An {@link com.airbnb.epoxy.EpoxyController} that meant for large lists of items.
+ * <p>
+ * Normally {@link #buildModels()} should build a unique model to represent every item in the
+ * RecyclerView; however, that can be slow if there are more than a few hundred items. This
+ * controller will manage your list for you and only request models to be built for the items in the
+ * list around the current position on screen. As the user scrolls models will be rebuilt around the
+ * scroll position as needed.
+ * <p>
+ * Additionally, this works with the Android Architecture component {@link PagedList} to load more
+ * items as needed.
+ * <p>
+ * To use, simply call {@link #setList} with either a normal Java {@link List} or an Android
+ * PagedList.
+ * <p>
+ * {@link #buildModels(List)} will be called with the subset of items that should have models built
+ * for them.
+ *
+ * @param <T> The type of item in the list
+ *
+ * @deprecated Use {@link PagedListEpoxyController} instead.
+ */
+@Deprecated()
```

Comment:
I think we can delete this file. It has been deprecated for a long time - I don't see a reason to bring it along to this new module, seems like a good time to finally delete it

Reply:
SGTM!